source/basic.tex

```diff
@@ -3750,6 +3750,8 @@
 \pnum
 \indextext{value!indeterminate}%
 \indextext{indeterminate value}%
+\indextext{value!erroneous}%
```

Comment:
I'm wondering whether we should make "erroneous value" a defined term and simply use \defnadjx.  That feels a lot more straightforward.

Reply:
I could do that too. If we're going to do that, I think we should make the same change for indeterminate values.

That's also why I've just perpetuated whatever is there right now; perhaps the conversion of both terms to `\denfadjx` could be done in a follow-up PR.

Reply:
Alright, so I've noticed that "indeterminate value" has both the `\defnadjx` definition and the `\indextext` up there.

Now, it's the same for erroneous values. Is there a reason why indeterminate values have both? Can/should we just get rid of all those `\indextext` entries?

Reply:
When removing `\indextext`, all index entries still exist, but we lose the link within the PDF and the page number.
![image](https://github.com/cplusplus/draft/assets/22040976/dba039ed-f70e-407f-9d94-505009c56480)

With `\indextext`, there's a `68` at the end.

Reply:
Yup, that's intentional: we want the "see" entry, not a direct page number.

Reply:
So, please remove \indextext and use \defnadjx